master/internal/core.go

```diff
@@ -832,7 +832,7 @@ func (m *Master) tryRestoreExperiment(sema chan struct{}, wg *sync.WaitGroup, e
 		if err := m.db.TerminateExperimentInRestart(e.ID, e.State); err != nil {
 			log.WithError(err).Error("failed to mark experiment as errored")
 		}
-		telemetry.ReportExperimentStateChanged(m.db, e)
+		telemetry.ReportExperimentStateChanged(context.TODO(), m.db, e)
```

Comment:
In this iteration, I have added `context.TODO()` to make it context configurable. I will complete this in next iteration.